src/NLog.Extensions.Logging/Internal/RegisterNLogLoggingProvider.cs

```diff
@@ -30,6 +33,24 @@ internal static void TryAddNLogLoggingProvider(this IServiceCollection services,
                 addLogging?.Invoke(services, (builder) => builder?.AddFilter<NLogLoggerProvider>(null, Microsoft.Extensions.Logging.LogLevel.Trace));
             }
         }
-    }
 #endif
+
+        internal static void TryLoadConfigurationFromSection(this NLogLoggerProvider loggerProvider, IConfiguration configuration)
+        {
+            if (!string.IsNullOrEmpty(loggerProvider.Options.LoadConfigurationFromSection))
```

Comment:
Maybe nice to revert the condition to prevent too much nesting

Reply:
Changed